src/fmain.pas

```diff
@@ -6774,34 +6892,36 @@ procedure Tfrmain.milistPropertiesClick(Sender: TObject);
           for i:=0 to frmain.lvMain.Items.Count-1 do
           begin
             //// Change only the normal download type and with pause, complete status
-            if (frmain.lvMain.Items[i].SubItems[columntype] = '0') and ((frmain.lvMain.Items[i].SubItems[columnstatus]='0') or (frmain.lvMain.Items[i].SubItems[columnstatus]='3')) and frmain.lvMain.Items[i].Selected then
+            itm := frmain.lvMain.Items[i];
+            if itm.Selected and (itm.SubItems[columntype] = '0') and ((itm.SubItems[columnstatus]='0') or (itm.SubItems[columnstatus]='3')) then
```

Comment:
I've placed `itm.Selected` as the first condition because, as far as I remember, Pascal uses left-to-right calculations (similar to C++), so it makes more sense to check first whether the item is selected than to check the item's additional states.

Reply:
yes and if itm.Selected is false then no continue checking conditions